shared/src/TlsVersions.cs

```diff
@@ -32,7 +31,7 @@ public class TlsVersions
         public SslProtocols Preferred { get; private set; } = SslProtocols.None;
 
 #if NET451
-        private SecurityProtocolType _net451Protocol = (SecurityProtocolType)PreferredProtocol;
+        private System.Net.SecurityProtocolType _net451Protocol = (System.Net.SecurityProtocolType)PreferredProtocol;
```

Comment:
Why do we want to move `System.Net` out of `using` directive?

Reply:
I just hated seeing it gray most of the time, tricking me out as if it were an unused using.